srcpkgs/git/template

```diff
@@ -23,6 +23,11 @@ make_check_target=test
 
 subpackages="git-cvs git-svn gitk git-gui git-all"
 
+case "$XBPS_TARGET_MACHINE" in
+*-musl) configure_args="${configure_args} ac_cv_fread_reads_directories=yes" ;;
+*) configure_args="${configure_args} ac_cv_fread_reads_directories=no" ;;
```

Comment:
One more thing: the two lines above should be indented with a tab.

Reply:
`configure_args="${configure_args} ac_...` can be shorted to `configure_args+=" ac_...`. But be sure to have one space before `ac`.

Reply:
Thanks @Johnnynator, I totally overlooked this though I suggested it this way above! ;)

Reply:
Sorry, I overlooked that detail.
New patch is on the way